components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -47,14 +48,15 @@ const ContributorListTableRow = ({
   topic,
   selected,
   handleOnSelectContributor,
+  range,
 }: ContributorListTableRow) => {
   const [tableOpen, setTableOpen] = useState(false);
 
   const { data: user } = useFetchUser(contributor.author_login);
-  const { data } = useContributorPullRequests(contributor.author_login, topic, [], 30);
+  const { data } = useContributorPullRequests(contributor.author_login, topic, [], range);
```

Comment:
For the list contributors page, I discovered that I wasn't passing the range filter